app/components/pagination/PaginationSelectionFooter.tsx

```diff
@@ -53,6 +64,7 @@ export default function PaginationSelectionFooter({
             page={page}
             limit={limit}
             totalPages={totalPages}
+            view={view}
```

Comment:
Why does the Pagination component need to know about the view?

Reply:
because in getPageLink if the view is not set as a search param, then when it creates the link, it doesn't include view. If it doesn't include view, then the view defaults to index. If it's always index, then the pagination links will never work for groups. To show you what I mean, here is what happens when I remove the view from getPageLinks. When I hover over the 2 button, this is the link. You will see that it doesn't include the view so when that page is navigated to, it's the index view which is the default.
<img width="839" alt="pagination_view" src="https://github.com/user-attachments/assets/40f3bd96-4a1f-4827-8014-faa35c2bffb1">

Reply:
I think that's a good idea. It did feel odd to have to add view specific code to it.